garg2021re/openreview/sections/results.tex

```diff
@@ -0,0 +1,181 @@
+\section{Results}
+In Table \autoref{tab:table1} we report our results for various classification benchmarks using our implemented BayesBiNN and STE optimizer. We notice that we get a difference of less than 0.1\% as compared to that in the original paper. We generated the results for baseline STE optimizer and full-precision networks by evaluating our implementation of these methods. We also generated the results of PMF, by modifying its original open-sourced code and using the hyperparameters mentioned in the original paper.
+
+\begin{figure}[h]
+     \centering
+     \begin{subfigure}[b]{0.3\textwidth}
+         \centering
+         \includegraphics[width=1.2\textwidth]{../openreview/figs/MNIST.png}
+         \caption{MNIST}
+     \end{subfigure}
+     \hfill
+     \begin{subfigure}[b]{0.3\textwidth}
+         \centering
+         \includegraphics[width=1.2\textwidth]{../openreview/figs/CIFAR10.png}
+         \caption{CIFAR10}
+     \end{subfigure}
+     \hfill
+     \begin{subfigure}[b]{0.3\textwidth}
+         \centering
+         \includegraphics[width=1.2\textwidth]{../openreview/figs/CIFAR100.png}
+         \caption{CIFAR100}
+     \end{subfigure}
+        \caption{Training/Validation/Test accuracy using BayesBiNN optimizer}
+\end{figure}
+
+\begin{table}[h]
+\begin{center}
+\begin{tabular}{ | c | c | c | c | c | }
+\hline
+ Datasets & Optimizer & Training Accuracy & Validation Accuracy & Test Accuracy \\ \hline
+  \multirow{5}{4em}{MNIST} 
+   & BayesBiNN(ours) & 99.90 \pm 0.01\% & 99.89 \pm 0.07\% & 98.87 \pm 0.06\%  \\
```

Comment:
Replace `\pm` with `$\pm$`. Overleaf doesn't complain because it tries to fix it in its backend, but its an incorrect practice (will save you a lot of time later in your future arxiv submissions!).